recipes/loky/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "loky" %}
+{% set version = "2.1.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d042ce3600212f510770777f5f6166d9fd5fe86002851b202fdece3830b6e5b1
+
+build:
+  number: 0
+  noarch: python
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - cloudpickle
+
+test:
+  imports:
+    - loky
+
+about:
+  home: https://github.com/tomMoral/loky
+  license: BSD 3-Clause
+  license_file: LICENSE.txt
+  summary: A robust implementation of concurrent.futures.ProcessPoolExecutor
+
+extra:
+  recipe-maintainers:
+    - jakirkham
```

Comment:
You can add me in the maintainers of the recipe, so I can help update when we release new version.

Reply:
Cool, thanks. Have added you.